src/Forms/Prism.Forms/Services/Dialogs/IDialogResult.cs

```diff
@@ -2,9 +2,20 @@
 
 namespace Prism.Services.Dialogs
 {
+    /// <summary>
+    /// Contains <see cref="IDialogParameters"/> from the dialog
+    /// and the <see cref="System.Exception"/> of the dialog.
+    /// </summary>
     public interface IDialogResult
     {
+        /// <summary>
+        /// The exception of the dialog.
```

Comment:
For Get/Set properties lets use the format of `Gets (or sets) the [insert description here]`. If no setter then do not add the "or sets" portion.

Reply:
I use the format as bellow:  
[https://github.com/PrismLibrary/Prism/pull/2761/files#diff-627f40b52b5622df540ef8100476c49cc13a491a5c1e17177ce2ab9a409d0e97R11](https://github.com/PrismLibrary/Prism/pull/2761/files#diff-627f40b52b5622df540ef8100476c49cc13a491a5c1e17177ce2ab9a409d0e97R11)  
https://github.com/PrismLibrary/Prism/pull/2761/files#diff-627f40b52b5622df540ef8100476c49cc13a491a5c1e17177ce2ab9a409d0e97R16